var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -65,7 +66,8 @@ class PyPillow(Package):
 
     # Required dependencies
     extends('python')
-    depends_on('binutils', type='build')
+    # Known not to work with 2.23, 2.25
+    depends_on('binutils@2.26:', type='build')
```

Comment:
Wait, so `py-pillow` doesn't work with `binutils@2.25`? Then how have people been building it all this time?